### PR TITLE
feat(shared-data): add peak shared data definition

### PR DESCRIPTION
**Pull request by ryanthecoder**
# Overview

This new pipette type features a different threaded rod drive nut and some other mechanical changes that increase the maximum speed that the plunger can move so that this pipette can be used in emulsification applications.

### CODE REVIEW

**Review by sfoster1 — CHANGES_REQUESTED** [review]
it is unfortunately `PEEK`, for the material

File: shared-data/command/schemas/11.json [hunk 1]
```diff
@@ -2690,6 +2690,7 @@
         "p1000_single_gen2",
         "p1000_single_flex",
         "p1000_multi_flex",
+        "p1000_multiemulsify",
```
Comment (sfoster1) [inline shared-data/command/schemas/11.json]:
```suggestion
        "p1000_multi_emulsify",
```

Comment (sfoster1) [reply shared-data/command/schemas/11.json]: I kind of hate this but if it makes everything a million times harder to have the underscore we're fine without it

**Review by sfoster1 — APPROVED** [review]
I'd love to figure out a way for this to be `p1000_multi_emulsify` (with an underscore) but if it makes it lots harder it's okay to not have it

File: shared-data/command/schemas/11.json (hunk 1 above)
Comment (ryanthecoder) [reply shared-data/command/schemas/11.json]: See the commit after this, I think this is actually the only place we need to have it specially processed.

**Review by sfoster1 — APPROVED** [review]
Okay, looks good to me!

**Pull request merged**